lib/features/quiz/screen/quiz_screen.dart

```diff
@@ -0,0 +1,169 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_learn_app/features/quiz/model/quiz.dart';
+import 'package:flutter_learn_app/features/theory/screens/theory_screen.dart';
+
+class QuizScreen extends StatefulWidget {
+  const QuizScreen({super.key});
+  @override
+  State<QuizScreen> createState() => _QuizScreenState();
+}
+
+class _QuizScreenState extends State<QuizScreen> {
+  List<Question> questionList = getQuestions();
+  int currentQuestionIndex = 0;
+  int score = 0;
+  Answer? selectedAnswer;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: const Color.fromARGB(255, 55, 50, 80),
+      body: Container(
+        margin: const EdgeInsets.symmetric(horizontal: 16, vertical: 32),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.spaceAround,
+          children: [
+            const Text(
+              'Quiz App',
+              style: TextStyle(color: Colors.white, fontSize: 24),
+            ),
+            _questionWidget(),
+            _answerList(),
+            _nextButton()
+          ],
+        ),
+      ),
+    );
+  }
+
+  _questionWidget() {
```

Comment:
bad idea to create widgets with a method, move it into separate class class _QuestionWidget, class _AnswerList etc

Reply:
and create class with const constructor, that is much effective

Reply:
ok